presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapAuthenticator.java

```diff
@@ -126,6 +131,31 @@ private Principal authenticate(String user, String password)
         }
     }
 
+    // Based on https://www.owasp.org/index.php/Preventing_LDAP_Injection_in_Java and javax.naming.ldap.Rdn.escapeStringValue
+    @VisibleForTesting
+    static boolean containsSpecialLDAPCharacters(String user)
```

Comment:
remove `LDAP` from method name. We're in scope of LDAP-specific class. 

add javadoc like
```
 /**
  * Returns {@code true} when parameter contains a character that has a special meaning in
  * LDAP search or bind name (DN).
 */
```